AmericanMedicalAssociation/ama-style-guide-2
# Adf 140 prototype interview questions form

**Jira Ticket**
- [ADF-140: Prototype "interview questions" form](https://palantir.atlassian.net/browse/ADF-140)

## Description
Created dr-finder-interview-questions molecule 

![Screen Shot 2022-10-12 at 9 09 56 AM](https://user-images.githubusercontent.com/89882376/195365843-03ce4213-f613-4a39-8ba8-af574f4bd99c.png)

## To Test

- Open your style guide by running `gulp serve`
- Navigate to `?p=pages-dr-finder-with-profile-edit` and look at desired fields (refer to the image below)
 ![image](https://user-images.githubusercontent.com/89882376/195365691-46426eb3-da76-40d1-8ba8-d8b3bbcb3834.png)

- Navigate to Molecules>Dr-Finder-Interview-Questions (Refer to the image below)
![Screen Shot 2022-10-12 at 9 15 39 AM](https://user-images.githubusercontent.com/89882376/195367167-81470470-f2f2-4b03-9b48-4d429b94c148.png)
- Ensure that all fields are identical to what is found in Pages>Dr-finder-with-profile-edit

 ## Relevant Screenshots/GIFs
 The section that we wanted to turn into a molecule
 ![image](https://user-images.githubusercontent.com/89882376/195365691-46426eb3-da76-40d1-8ba8-d8b3bbcb3834.png)


[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
